static/app/__mocks__/api.tsx

```diff
@@ -263,11 +264,13 @@ class Client implements ApiNamespace.Client {
         response.callCount++;
 
         const errorResponse = Object.assign(
-          {
+          new RequestError(options.method || 'GET', url, new Error(), {
```

Comment:
the mock response wasn’t rejecting with a `RequestError` like the production does; this should narrow the gap

Reply:
I will live long enough to see this mock client be removed one day :)